ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/ExceptionRequestTableCells.tsx

```diff
@@ -142,9 +142,12 @@ export type RequestScopeProps = {
 };
 
 export function RequestScope({ scope }: RequestScopeProps) {
-    return (
-        <div>{`${scope.imageScope.registry}/${scope.imageScope.remote}:${scope.imageScope.tag}`}</div>
-    );
+    const { registry, remote, tag } = scope.imageScope;
+    const text =
+        registry === '.*' && registry === '.*' && registry === '.*'
```

Comment:
Should each piece of the conditional be checking a different property?
```suggestion
        registry === '.*' && remote === '.*' && tag === '.*'
```

Reply:
Oh my, yes that is what I meant. Should have double checked the variables 🤦🏼